voicevox_engine/morphing.py

```diff
@@ -193,6 +198,9 @@ def synthesis_morphing(
         morph_param.frame_period,
     )
 
+    if output_fs != morph_param.fs:
+        y_h = resample(y_h, output_fs * len(y_h) // morph_param.fs)
+
```

Comment:
リサンプル処理がsynthesis_engine.pyにあるものと一緒なので共通化したほうが良いかも、と思ったのですが、良い方法を思いつきませんでした･･･。
`output_stereo`や`output_fs`などのパラメータを受け取って後処理をするだけの便利関数を作る･･･？うーん。。
そのままでも良いかなと思いました 😇

Reply:
TODOコメントを残しても良いかなと思いました。
```suggestion
    # TODO: synthesis_engine.py でのリサンプル処理と共通化する
    if output_fs != morph_param.fs:
        y_h = resample(y_h, output_fs * len(y_h) // morph_param.fs)

```